Mathlib/Data/ZMod/Basic.lean

```diff
@@ -1372,6 +1372,25 @@ end lift
 
 end ZMod
 
+section Module
+variable {S G : Type*} [AddCommGroup G] {n : ℕ} [Module (ZMod n) G] [SetLike S G]
+  [AddSubgroupClass S G] {K : S} {x : G}
+
+lemma zmod_smul_mem (hx : x ∈ K) : ∀ a : ZMod n, a • x ∈ K := by
```

Comment:
```suggestion
lemma zmod_smul_mem (a : ZMod n) (hx : x ∈ K) : a • x ∈ K := by
```
seems like a better argument order.

Alternatively, can you create a `SMulMemClass` instance?

Reply:
I wrote `zmod_smul_mem` to match `nsmul_mem`/`zsmul_mem`. These disagree with your argument order.

The `SMulMemClass` statement cannot be made an instance because of `R := ZMod n` being an out-param and the `Module (ZMod n) G` argument. I added it as a lemma instead